python/RappCloud/CloudMsgs/ObjectDetectionFindObjects.py

```diff
@@ -0,0 +1,89 @@
+from RappCloud.Objects import (
+    File,
+    Payload)
+
+from Cloud import (
+    CloudMsg,
+    CloudRequest,
+    CloudResponse)
+
+
+class ObjectDetectionFindObjects(CloudMsg):
+    """ Object Detection - Find Objects CloudMsg object"""
+
+    class Request(CloudRequest):
+        """ Object Detection - Find Objects Cloud Request object. ObjectDetectionFindObjects.Request """
+        def __init__(self, **kwargs):
+            """!
+            Constructor
+
+            @param **kwargs - Keyword arguments. Apply values to the request attributes.
+                - @ref imageFilepath
+                - @ref fast
+            """
+
+            ## File path to the image to load. This is the image to perform
+            # object detection on.
+            self.fname = ''
+            ## Limit results to this number of objects
+            self.limit = 1
+            ## Username 
+            self.user = 'rapp'
```

Comment:
To remove. User information is retrieved on the RAPP Platform using the user's credentials (token).
Resource [here](https://github.com/rapp-project/rapp-platform/wiki/How-to-create-a-HOP-service-for-a-ROS-service%3F#web-service-implementation)